shell/browser/extensions/api/runtime/electron_runtime_api_delegate.cc

```diff
@@ -43,7 +43,7 @@ bool ElectronRuntimeAPIDelegate::CheckForUpdates(
 void ElectronRuntimeAPIDelegate::OpenURL(const GURL& uninstall_url) {}
 
 bool ElectronRuntimeAPIDelegate::GetPlatformInfo(PlatformInfo* info) {
-  const char* os = update_client::UpdateQueryParams::GetOS();
+  const char* const os = update_client::UpdateQueryParams::GetOS();
```

Comment:
I don't think this is really needed for local variables as they don't present any risk of compiling to a static initializer.